Exiled.Events/Patches/Events/Player/ChangingRole.cs

```diff
@@ -82,13 +82,14 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 // Handlers.Player.OnChangingRole(ev)
                 new(OpCodes.Call, Method(typeof(Player), nameof(Player.OnChangingRole))),
 
-                new(OpCodes.Callvirt, Method(typeof(ChangingRoleEventArgs), nameof(ChangingRoleEventArgs.ConsolidatePlayerInformation))),
-
                 // if (!ev.IsAllowed)
                 //    return;
                 new(OpCodes.Callvirt, PropertyGetter(typeof(ChangingRoleEventArgs), nameof(ChangingRoleEventArgs.IsAllowed))),
                 new(OpCodes.Brfalse_S, returnLabel),
 
+                // PlayerRoles being updated.
+                new(OpCodes.Callvirt, Method(typeof(ChangingRoleEventArgs), nameof(ChangingRoleEventArgs.ConsolidatePlayerInformation))),
+
```

Comment:
Check again

Reply:
![image](https://user-images.githubusercontent.com/24619207/176468997-a5e76b9b-91bc-48a9-907d-6183a4baf6ad.png)

??? git ???

Reply:
![image](https://user-images.githubusercontent.com/24619207/176469043-f6a1fbdc-c600-4a14-8035-5dc7c3ac4796.png)

Reply:
😈

Reply:
I don't know how that happened but OK - sorry I guess lmao

Reply:
I did, I think github was freaking out because I kept pushing changes every minutes, I kept refreshing and it didn't show my previous commit until a few minutes later, way after you make this review. Very weird

Reply:
Also I've had 8 hours of sleep in the last two days, my brain a little fried so thanks for the help @louis1706 @IRacle1

Reply:
don't worry.

Reply:
But the problem with git is really weird, I've never had that